fix platform version in podspec

soranoba opened the pull request:
Thank you for a great library.
I found a warning on iOS, so I fixed it.

NOTE: Platform version in React.podspec is iOS9.0+.
https://github.com/facebook/react-native/blob/0926373c9f6e5191fe2253534576ea8a9fd3c7ab/React.podspec#L41


![2018-08-16 2 40 07](https://user-images.githubusercontent.com/4954289/44163538-b85c6380-a0fe-11e8-84d4-e65417a4215f.png)

hampustagerud commented:
I'm guessing this is part of the FontAwesome5 part of the library? I updated the way it works in #808 and effectively removed the need for these variables to be used. I missed to update the getImageSource() function to support this so thank you for bringing this to attention 🎉, I have just updated it which would remove this warning.

I don't know if Apple allows apps to be submitted with a deployment target of <9.0. While it might not break too many apps I think the entire project should at least be updated to the same target, not just the podfile 🙂

soranoba commented:
> I'm guessing this is part of the FontAwesome5 part of the library? I updated the way it works in #808 and effectively removed the need for these variables to be used. I missed to update the getImageSource() function to support this so thank you for bringing this to attention 🎉, I have just updated it which would remove this warning.

I confirmed that this PR is included in #808 .
Thank you for teaching.
However, since I can not judge the validity of the contents of #808, I would like the maintainer to judge.
(If @hampustagerud is a maintainer, please close this PR 🙇 )

> I don't know if Apple allows apps to be submitted with a deployment target of <9.0. While it might not break too many apps I think the entire project should at least be updated to the same target, not just the podfile 🙂

Since the current ReactNative supports iOS 9~, we can not release it under iOS 9 as long as we use the latest.
And, because their warnings may indicate undefined constant, it may already broken on iOS8.

hampustagerud commented:
I don't judge this PR, I just wanted to tell about the changes I've made which are not incorporated yet 🙂

soranoba commented:
👍 
bump to ios9.

hampustagerud commented:
I have no issue with requiring iOS >= 9 🙂
